src/WebJobs.Script/Workers/ProcessManagement/WorkerProcess.cs

```diff
@@ -168,16 +130,38 @@ private void OnOutputDataReceived(object sender, DataReceivedEventArgs e)
         {
             if (e.Data != null)
             {
-                string msg = e.Data;
-                if (WorkerProcessUtilities.IsConsoleLog(msg))
-                {
-                    msg = WorkerProcessUtilities.RemoveLogPrefix(msg);
-                    _workerProcessLogger?.LogInformation(msg);
-                }
-                else
-                {
-                    _consoleLogSource?.Log(msg);
-                }
+                BuildAndLogConsoleLog(e.Data, LogLevel.Information);
+            }
+        }
+
+        private void BuildAndLogConsoleLog(string msg, LogLevel level = LogLevel.Information)
+        {
+            ConsoleLog consoleLog = new ConsoleLog()
+            {
+                Message = msg,
+                Level = level
+            };
+
+            if (msg.IndexOf("warn", StringComparison.OrdinalIgnoreCase) > -1)
+            {
+                consoleLog.Level = LogLevel.Warning;
```

Comment:
I think we only want to assign "warn" if this came from stderr output?

Reply:
fixed!